src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/postponementrequest/PostponementRequestAboutToStartHandlerTest.java

```diff
@@ -125,5 +127,17 @@ public void givenAPostponementRequest_setupPostponementRequestData() {
         assertThat(response.getData().getPostponementRequest().getPostponementRequestHearingDateAndTime(), is(hearing.getValue().getHearingDateTime().toString()));
     }
 
+    @Test
+    public void givenAPostponementRequestfromGaps_returnAnError() {
+        sscsCaseData.setRegionalProcessingCenter(RegionalProcessingCenter.builder().hearingRoute(HearingRoute.GAPS).build());
+        final PreSubmitCallbackResponse<SscsCaseData> response = handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
+        assertThat(response.getErrors().size(), is(1));
+    }
 
+    @Test
+    public void givenAPostponementRequestfromListAssist_returnAnError() {
```

Comment:
Q: should this be `_shouldNotReturnAnError`?

Reply:
+1